roles/configure-nexus/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+
+- name: "Look up Nexus URL"
+  command: >
+    oc get route {{ nexus_name | default('nexus') }} -n {{ nexus_project }} -o custom-columns=ROUTE:.spec.host --no-headers
+  register: nexus_url_result
+  when:
+  - nexus_url is not defined or nexus_url|trim == ''
+
+- name: "Set nexus_url Result"
+  set_fact:
+    nexus_url: "{{ nexus_url_result.stdout }}"
+  when:
+  - nexus_url is not defined or nexus_url|trim == ''
+
+- name: "Ensure Nexus is available before attempting to communicate"
+  import_tasks: wait-for-nexus.yml
```

Comment:
Did `wait-for-nexus.yml` get lost in the merge? It's missing and the role crashes

Reply:
fixed

Reply:
it was due to the `.gitignore` changing a long the way (since this PR has been in progress for awhile)